pandas-stubs/core/groupby/generic.pyi

```diff
@@ -159,6 +159,10 @@ class DataFrameGroupBy(GroupBy, Generic[ByT]):
     def apply(  # pyright: ignore[reportOverlappingOverload]
         self, func: Callable[[Iterable], float], *args, **kwargs
     ) -> DataFrame: ...
+    # error: overload 1 overlaps overload 2 as "size" is in AggFuncTypeFrame
```

Comment:
Better to say `# error: overload 1 overlaps overload 2 because of different return types`